databricks/koalas/series.py

```diff
@@ -4324,6 +4324,58 @@ def pct_change(self, periods=1):
 
         return self._with_new_scol((scol - prev_row) / prev_row)
 
+    def combine_first(self, other):
+        """
+        Combine Series values, choosing the calling Series's values first.
+
+        .. note:: This API internally performs a join operation which can be pretty expensive
+            in general. if you want to use though, set `compute.ops_on_diff_frames` to True.
```

Comment:
I think it's fine to don't respect `compute.ops_on_diff_frames` when users call this API. There's no way to use joining to combine two dataframes.

Reply:
You can enable it via `option_context(...)`.